### PR TITLE
Added `setup_oscap` method for sat6 installation.

### PR DESCRIPTION
**Pull request by kbidarkar**
*) Configures the satellite6 with oscap packages, so that
policies, scap content and reports page is visible.

### CODE REVIEW

**Comment by Ichimonji10** [comment]
ACK pending Jenkins.

    $ flake8 .
    ./automation_tools/__init__.py:422:1: E302 expected 2 blank lines, found 1
    ./automation_tools/__init__.py:442:1: E302 expected 2 blank lines, found 1

Do we have any way of checking this change before merging it?

**Comment by kbidarkar** [comment]
looks like I missed to push the updated commit. Will push it soon.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -420,7 +420,28 @@ def setup_abrt():
...
+    packages = [
+        'rubygem-smart_proxy_openscap',
+        'ruby193-rubygem-foreman_openscap'
+    ]
+    for package in packages:
+        run('yum install -y {0}'.format(package))
+
+    manage_daemon('restart', 'foreman')
+    manage_daemon('restart', 'httpd')
+    manage_daemon('restart', 'foreman-proxy')
```
Comment (Ichimonji10) [inline automation_tools/__init__.py]: One more request: can we use some loops?

```python
    for package in (
            'rubygem-smart_proxy_openscap',
            'ruby193-rubygem-foreman_openscap'):
        run('yum install -y {0}'.format(package))
    for daemon in ('foreman', 'httpd', 'foreman-proxy'):
        manage_daemon('restart', daemon)
```

**Comment by Ichimonji10** [comment]
Comment would be nice to have addressed. However, it's a fairly light refactor, so don't block if this PR is high priority.

File: automation_tools/__init__.py (hunk 1 above)
Comment (kbidarkar) [inline automation_tools/__init__.py]: Ok, I will change it for the daemon , but not for packages, it's just fine and also for consistency sake.

**Comment by jyejare** [comment]
ACK pending @Ichimonji10 's comments.

**Comment by elyezer** [comment]
:+1:

**Pull request merged**